taxi/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="Test Manufacturer")
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="Test Driver",
+            first_name="test first",
+            last_name="test last",
+            password="test123",
+        )
+        self.assertEqual(
+            str(driver),
+            f"{driver.username} ({driver.first_name} {driver.last_name})"
+        )
+
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(name="Test Manufacturer")
+        car = Car.objects.create(model="Test Model", manufacturer=manufacturer)
+        self.assertEqual(str(car), car.model)
+
+    def test_creat_driver_with_license(self):
```

Comment:
There's a typo in the method name `test_creat_driver_with_license`. It should be `test_create_driver_with_license`.